composer.json

```diff
@@ -13,14 +13,14 @@
         }
     },
     "require": {
-        "php": ">=5.5",
-        "zendframework/zend-eventmanager": "^2.6|^3.0"
+        "php": "^5.5 || ^7.0",
+        "zendframework/zend-eventmanager": "^2.6 || ^3.0"
     },
     "require-dev": {
         "doctrine/annotations": "~1.0",
-        "zendframework/zend-stdlib": "~2.7",
+        "zendframework/zend-stdlib": "^2.7 || ^3.0",
         "fabpot/php-cs-fixer": "1.7.*",
-        "phpunit/PHPUnit": "~4.0"
+        "phpunit/PHPUnit": "^4.5"
```

Comment:
This is for the 2.6 series; I *think* this is likely sufficient, but I can bump later if desired. (I see master uses 4.8.21 or something similar.)